test/mailers/previews/user_mailer_preview.rb

```diff
@@ -408,10 +408,15 @@ def admin_hidden_work_notification
     UserMailer.admin_hidden_work_notification(works.map(&:id), user.id)
   end
 
-  # URL: /rails/mailers/user_mailer/admin_deleted_work_notification
+  # Sent to a user when an admin deletes their work
+  # URL: /rails/mailers/user_mailer/admin_deleted_work_notification?work_id=2
   def admin_deleted_work_notification
     work = create(:work)
     user = create(:user, :for_mailer_preview)
```

Comment:
It'd be a bit cleaner to only create the user and work if no `work_id` is specified